website/content/cookbook/auto-tls.md

```diff
@@ -24,7 +24,3 @@ message with TLS enabled on the website.
 {{< embed "auto-tls/server.go" >}}
 
 ## [Source Code]({{< source "auto-tls" >}})
-
```

Comment:
Why remove the maintainer sections? 
At least for some middlewares they could be correct. But I'm not sure either.
Maybe it's good to start fresh here.

Reply:
with exception of vishr  name none of those names are actively maintaining those middleware and vishr is on sabbatical. No point to have block that adds no value and to be correct needs to be regularly updated.

Reply:
You are right. No point in keeping non-active maintainers.